packages/extension/src/companion/suggestion/MessageSuggestion.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useThreadPageObserver } from './useThreadPageObserver';
+import { MessageSuggestionPopup } from './MessageSuggestionPopup';
+
+const bubbleClass = 'msg-overlay-conversation-bubble';
+const titleClass = 'msg-overlay-bubble-header__title';
+
+interface MessagePopup {
+  id: string;
+  bubble: HTMLElement;
+}
+
+export function MessageSuggestion() {
+  useThreadPageObserver();
+  const [popups, setPopups] = useState<MessagePopup[]>([]);
+  const [observer] = useState(
+    new MutationObserver(async () => {
+      await new Promise((resolve) => setTimeout(resolve, 1)); // wait for the dom to update
```

Comment:
```suggestion
      await new Promise((resolve) => setTimeout(resolve, 10)); // wait for the dom to update
```
just add a bit more timeout